docs/root/start/_include/dockerhub-images.csv

```diff
@@ -0,0 +1,9 @@
+,,stable,stable,master,master
+Docker image,description,amd64,arm64,amd64,arm64
+`envoyproxy/envoy <https://hub.docker.com/r/envoyproxy/envoy/tags/>`_,Release binary with symbols stripped on top of an Ubuntu Bionic base.,v1.16-latest,v1.16-latest,,
```

Comment:
WDYT about documenting the release vs. dev distinction here in the table also to make it super clear? As part of the description? (Basically moving or duplicating the below note "In the above dev repositories, the latest tag points to a container including the last Envoy build on master that passed tests." into here?)

Also, can you ping me on Slack with the text you want in the description for each repo and I will update it?

Reply:
> WDYT about documenting the release vs. dev distinction 

that its a bad idea.

afaict they are the same images (built with the same recipe) with the same binaries inside (built with same compiler) just at different commit points

they dont do anything different to each other

Reply:
in fact the note about master being release-quality kinda says as much

Reply:
OK I don't really have a strong opinion. In general I don't want to let perfect be the enemy of the good. This is a massive improvement so we can ship and iterate.

Reply:
ive readded the not about `X.Y-latest` versions and some info about the build image in the note under the table